lepotato/README

```diff
@@ -0,0 +1,6 @@
+The u-boot binary needs some tweaks before installing it, this is described in
```

Comment:
@fdanis-oss The only blocker I see for this MR is to clarify what this means ? can we "tweak" the binary and install u-boot to the SD card inside the recipe?

Reply:
@obbardc Amlogic does not provide sources for the firmware and tools needed to create a
bootloader image so it is necessary to use some proprietary binaries. This is
described in [u-boot-amlogic documentation](https://github.com/u-boot/u-boot/blob/master/doc/board/amlogic/libretech-cc.rst).  
I didn't embed this in a debos action as binaries licenses have not clear licenses, as pointed by @sjoerdsimons in first comment.

I updated the README, move recipe to use Debian `bookworm` and add an action to extract the `u-boot.bin` file from the installed `u-boot-amlogic` package, as it is needed to create the bootloader image.